platforms/osx/src/main.mm

```diff
@@ -13,11 +13,9 @@
 
 std::string sceneFile = "scene.yaml";
 
-std::string markerStyling = "{ style: 'points', interactive: true, color: 'white', size: [25px, 25px], order: 100, collide: false }";
+std::string markerStyling = "touch.point.icons";
```

Comment:
I can see how one might guess from a function name how 'layer' is implied, but guessing is not a good developer experience. Also, the comments above the function seem to directly contradict that implication, describing the argument as `the absolute path to a draw group in the scene`.